Source/include/Ably/ARTAuth+Private.h

```diff
@@ -0,0 +1 @@
+../../../Source/ARTAuth+Private.h
```

Comment:
As discussed in the meeting, these types of files (ending in `Private.h` should not be in the `include` directory.

Reply:
done, defined in module as Private